Tasks/PublishSymbols/make.json

```diff
@@ -0,0 +1,61 @@
+{
+    "nugetv2": [
```

Comment:
i added this directly as a top-level property instead of nested within an "externals" property. again i'm fine either way, this just felt a littel cleaner.

i still need to test out downloading and extracting zip files, which would be under a top-level property "archivePackages" which is an array of { url: string, dest: string }